Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1054,23 +1070,27 @@ theorem enorm_ae_le_eLpNormEssSup {_ : MeasurableSpace α} (f : α → ε) (μ :
 coe_nnnorm_ae_le_eLpNormEssSup := enorm_ae_le_eLpNormEssSup
 
 @[simp]
-theorem eLpNormEssSup_eq_zero_iff {f : α → F} : eLpNormEssSup f μ = 0 ↔ f =ᵐ[μ] 0 := by
+theorem eLpNormEssSup_eq_zero_iff {ε} [TopologicalSpace ε] [ENormedAddMonoid ε] {f : α → ε} :
+    eLpNormEssSup f μ = 0 ↔ f =ᵐ[μ] 0 := by
   simp [EventuallyEq, eLpNormEssSup_eq_essSup_enorm]
 
-theorem eLpNorm_eq_zero_iff {f : α → E} (hf : AEStronglyMeasurable f μ) (h0 : p ≠ 0) :
-    eLpNorm f p μ = 0 ↔ f =ᵐ[μ] 0 := by
+theorem eLpNorm_eq_zero_iff {ε} [TopologicalSpace ε] [ENormedAddMonoid ε] {f : α → E}
```

Comment:
Do you really want to keep `f` with values in `E`, or should it be in epsilon? If you fix this, I guess you can remove several weird type annotations from the PR.

Reply:
Good catch! Fixing this required a few more generalisations, but that should be nicer indeed.